sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/Diagnostics/AzureMonitorExporterEventSource.New.cs

```diff
@@ -82,5 +82,83 @@ public void StatsbeatFailed(Exception ex)
+        public void FailedToParseConnectionString(string exceptionMessage) => WriteEvent(12, exceptionMessage);
+
+        [Event(13, Message = "Unsupported Metric Type '{0}' cannot be exported.", Level = EventLevel.Warning)]
+        public void UnsupportedMetricType(string metricTypeName)
+        {
+            if (IsEnabled(EventLevel.Warning))
+            {
+                WriteEvent(13, metricTypeName);
+            }
+        }
+
+        [NonEvent]
+        public void FailedToConvertLogRecord(Exception ex)
+        {
+            if (IsEnabled(EventLevel.Error))
+            {
+                FailedToConvertLogRecord(ex.FlattenException().ToInvariantString());
+            }
+        }
+
+        [Event(14, Message = "Failed to convert Log Record due to an exception: {0}", Level = EventLevel.Error)]
+        public void FailedToConvertLogRecord(string exceptionMessage) => WriteEvent(14, exceptionMessage);
+
+        [NonEvent]
+        public void FailedToConvertMetricPoint(string meterName, string instrumentName, Exception ex)
+        {
+            if (IsEnabled(EventLevel.Error))
+            {
+                FailedToConvertMetricPoint(meterName, instrumentName, ex.FlattenException().ToInvariantString());
+            }
+        }
+
+        [Event(15, Message = "Failed to convert Metric Point due to an exception. MeterName: {0}. InstrumentName: {1}. {2}", Level = EventLevel.Error)]
+        public void FailedToConvertMetricPoint(string meterName, string instrumentName, string exceptionMessage) => WriteEvent(15, meterName, instrumentName, exceptionMessage);
+
+        [NonEvent]
+        public void FailedToConvertActivity(string activityDisplayName, Exception ex)
+        {
+            if (IsEnabled(EventLevel.Error))
+            {
+                FailedToConvertActivity(activityDisplayName, ex.FlattenException().ToInvariantString());
+            }
+        }
+
+        [Event(16, Message = "Failed to convert Activity due to an exception. Activity: {0}. {1}", Level = EventLevel.Error)]
+        public void FailedToConvertActivity(string activityDisplayName, string exceptionMessage) => WriteEvent(16, activityDisplayName, exceptionMessage);
+
+        [NonEvent]
+        public void FailedToExtractActivityEvent(string activityDisplayName, Exception ex)
+        {
+            if (IsEnabled(EventLevel.Error))
+            {
+                FailedToExtractActivityEvent(activityDisplayName, ex.FlattenException().ToInvariantString());
+            }
+        }
+
+        [Event(17, Message = "Failed to extract Activity Event due to an exception. Activity: {0}. {1}", Level = EventLevel.Error)]
+        public void FailedToExtractActivityEvent(string activityDisplayName, string exceptionMessage) => WriteEvent(17, activityDisplayName, exceptionMessage);
+
+        [Event(18, Message = "Maximum count of {0} Activity Links reached. Activity: {1}. ", Level = EventLevel.Informational)]
```

Comment:
Should this be a warning or information?

Reply:
Please make the message more user friendly/actionable.
What happened? What is the impact (do we drop whole span or whole links or just the excess links)? How to mitigate (if applicable)